packages/aila/src/core/quiz/LessonSlugQuizMapping.test.ts

```diff
@@ -1,170 +1,178 @@
 import { ElasticLessonQuizLookup } from "./LessonSlugQuizMapping";
 
-describe("ElasticLessonQuizLookup", () => {
-  let dbLookup: ElasticLessonQuizLookup;
+if (process.env.TEST_QUIZZES === "true") {
```

Comment:
Selectively turns off these quiz tests until we have a way of inserting test data into the database

Reply:
I would take the `!== "false"` approach from below so that you have to opt into this behaviour. Otherwise we risk having the tests disabled on staging

```suggestion
if (process.env.TEST_QUIZZES !== "false") {
```

Another approach could be to add a new pnpm command like `pnpm test:no-quizzes` for local dev

Reply:
I'm also trying to think of a way to avoid this large diff. Have you tried:

```
const maybeDescribe = process.env.TEST_QUIZZES === "false" ? describe.skip : describe
```

Reply:
Ah that's a good idea. Thanks!

Reply:
I've managed to find a way to make that work in a similar way to what you've suggested, thanks